extensions/README.md

```diff
@@ -56,7 +56,7 @@ with the template.
 
 The logic for when an object should list an extension ID in its `stac_extension` array is as follows:
 
-- If the object directly implements the extension, the  `stac_extensions` of that object should contain the extension ID.
+- If the object directly implements the extension (by including at least one of the fields of the extension, plus any additional specified requirements), the  `stac_extensions` of that object should contain the extension ID.
```

Comment:
As there may be some extensions that don't add fields, maybe say something like:

> by including at least one of the fields of the extension (if there are any) or implementing proposed behaviors, plus any additional specified requirements.

Please improve the wording that seems like a very "German" way of writing it...

Reply:
Will there be extensions that don't add fields? What would they do?

If there are ones that don't add fields then I think I might just say '(by including the fields specified by the extension)'